db/migrations/schema-objects/20220118152509.do.audit-operation-type-lookup-table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE [mtc_admin].[auditOperationTypeLookup](
+	[id] [int] IDENTITY(1,1) NOT NULL,
+	[createdAt] [datetimeoffset](3) NOT NULL,
+	[updatedAt] [datetimeoffset](3) NOT NULL,
+	[version] [timestamp] NOT NULL,
+	[auditOperation] [nvarchar](255) NOT NULL,
```

Comment:
If we plan to pull the PK ID field based on the 'auditOperation' column then 'auditOperation' ought to be UNIQUE.